### DISCUSSION TITLE
failed to dial websocket 求助大神

### DISCUSSION THREAD

**Discussion by desirenathan** [v2fly/v2ray-core#2096] [General]
2022/10/24 12:24:00 [Warning] [3838160291] v2ray.com/core/app/proxyman/outbound: failed to process outbound traffic 
    > v2ray.com/core/proxy/vmess/outbound: failed to find an available destination 
    > v2ray.com/core/common/retry: [v2ray.com/core/transport/internet/websocket: failed to dial WebSocket 
    > v2ray.com/core/transport/internet/websocket: failed to dial to (wss://vps.tibetancordyceps.com/ray):  > read tcp 192.168.1.5:55509->23.105.211.197:443: wsarecv: An existing connection was forcibly closed by the remote host. v2ray.com/core/transport/internet/websocket: failed to dial WebSocket 
    > v2ray.com/core/transport/internet/websocket: failed to dial to (wss://vps.tibetancordyceps.com/ray):  > read tcp 192.168.1.5:55511->23.105.211.197:443: wsarecv: An existing connection was forcibly closed by the remote host. v2ray.com/core/transport/internet/websocket: failed to dial WebSocket 
    > v2ray.com/core/transport/internet/websocket: failed to dial to (wss://vps.tibetancordyceps.com/ray):  > read tcp 192.168.1.5:55517->23.105.211.197:443: wsarecv: An existing connection was forcibly closed by the remote host. v2ray.com/core/transport/internet/websocket: failed to dial WebSocket 
    > v2ray.com/core/transport/internet/websocket: failed to dial to (wss://vps.tibetancordyceps.com/ray):  > read tcp 192.168.1.5:55518->23.105.211.197:443: wsarecv: An existing connection was forcibly closed by the remote host. v2ray.com/core/transport/internet/websocket: failed to dial WebSocket 
    > v2ray.com/core/transport/internet/websocket: failed to dial to (wss://vps.tibetancordyceps.com/ray):  > read tcp 192.168.1.5:55519->23.105.211.197:443: wsarecv: An existing connection was forcibly closed by the remote host.] 
    > v2ray.com/core/common/retry: all retry attempts failed

**Comment by web-elite** [comment]
i have same problem ...

**Comment by wpblank** [comment]
客户端不要设置 alpn

**Reply by cnscorpions** [reply]
老哥，我使用的 v2rayN v3.29 版本，找不到这个配置项?

**Comment by tiddler83** [comment]
一模一样的问题。
奇怪的是，上周使用移动的数据网络时没问题，但本周开始出现这个问题。而使用电信的宽带和数据都没问题。
移动和电信采用的GFW策略不同？

**Reply by fatgenius** [reply]
我也是遇到了

**Comment by objdjm** [comment]
我也遇到了，卡了两周没解决

**Reply by trappedlamb** [reply]
怎么解决的呢？现在租的是移动宽带出现了这个问题

**Comment by lza1234567** [comment]
请问你解决了没

**Comment by desirenathan** [comment]
没有，最后向机场屈服了。
________________________________
